modules/rosa-common-commands.adoc

```diff
@@ -19,6 +19,17 @@ Enables debug mode for the parent command.
 $ rosa create cluster --cluster=<cluster_name> --debug
 ----
 
+[id="rosa-download_{context}"]
+== download
+
+Downloads the latest compatible version of the specified software to the current directory in an archive file. Extract the contents of the archive and add the contents to your path to use the software. To download the latest Red Hat OpenShift Service on AWS CLI, specify `rosa`. To download the latest OpenShift Container Platform CLI, specify `oc`.
+
+.Examples
```

Comment:
Should this be singular if it's just a single example?

Reply:
Yes, thankyou!